### PR TITLE
Fix bug in load audio (multi-channel)

### PR DESCRIPTION
**Pull request by desh2608**

### CODE REVIEW

**Comment by pzelasko** [comment]
Thanks! Could you also add a test for this?

**Comment by desh2608** [comment]
> Thanks! Could you also add a test for this?

Done.

File: lhotse/features/base.py [hunk 1]
```diff
@@ -446,6 +446,7 @@ def load(
         self,
         start: Optional[Seconds] = None,
         duration: Optional[Seconds] = None,
+        channel_id: Union[int, List[int]] = 0,
```
Comment (desh2608) [inline lhotse/features/base.py:449]: This should address https://github.com/lhotse-speech/lhotse/issues/908, but it is currently unused in the function.

**Comment by desh2608** [comment]
> Thanks! Could you also add a test for this?

I have added tests, and also a small fix for the MultiCut feature loading issue (although it is not a complete fix at this point).

**Review by pzelasko — APPROVED** [review]

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**